ci/gpu/build.sh

```diff
@@ -54,14 +54,21 @@ conda install -c conda-forge -c rapidsai -c rapidsai-nightly -c rapidsai/label/x
       "umap-learn" \
       "protobuf >=3.4.1,<4.0.0" \
       "nccl>=2.4" \
-      "dask=2.8*" \
-      "distributed=2.8*" \
+      "dask>=2.8.0" \
+      "distributed>=2.8.0" \
       "dask-ml" \
       "dask-cudf=${MINOR_VERSION}" \
       "dask-cuda=${MINOR_VERSION}" \
       "statsmodels" \
       "xgboost=0.90.rapidsdev1"
```

Comment:
Subsequent change - we should switch to using rapids-xgboost=1.0.0

Reply:
I think this should be a separate PR so that it is dedicated to testing the new xgboost package